client_test/integration/cases.go

```diff
@@ -0,0 +1,21 @@
+package integration_test
+
+import "github.com/statechannels/go-nitro/internal/testactors"
+
+var simpleCase = TestCase{
```

Comment:
TODO: Add some simple validation that makes sure all test cases values are "valid". For example, right now we only support up to 2 hops.